src/modules/user/dto/createUser.dto.ts

```diff
@@ -1,14 +1,18 @@
 import { IsEmail, IsNotEmpty, MinLength } from "class-validator";
 import { EmailIsUnique } from "../validation/email-is-unique.validator";
+import { ApiProperty } from "@nestjs/swagger";
```

Comment:
qq isso faz, to curioso

Reply:
esse decorator coloca o campo na documentação do swagger
o que precisa pra executar a request :fire: 

![image](https://user-images.githubusercontent.com/55204419/226625190-0bdebfa3-fc28-4d71-b650-41f409c38854.png)

tem muitas opções para escolher, mas ainda não é a hora